client/src/App.css

```diff
@@ -59,3 +59,6 @@ main {
 footer {
   text-align: center;
 }
+.tryText {
```

Comment:
Generally it's good practice to keep CSS classnames in "kebab case" like `.try-text`. And don't forget to change the class attribute in the HTML as well to match! :+1: